src/test/java/io/cdap/plugin/servicenow/source/ServiceNowMultiSourceTest.java

```diff
@@ -16,147 +16,212 @@
+      "            \"notification\": \"2\",\n" +
+      "            \"enable_multifactor_authn\": \"false\",\n" +
+      "            \"sys_updated_by\": \"system\",\n" +
+      "            \"sys_created_on\": \"2019-04-05 21:09:12\",\n" +
+      "            \"sys_domain\": {\n" +
+      "                \"link\": \"https://ven05127.service-now.com/api/now/table/sys_user_group/global\",\n" +
+      "                \"value\": \"global\"\n" +
+      "            },\n" +
+      "            \"state\": \"\",\n" +
+      "            \"vip\": \"false\",\n" +
+      "            \"sys_created_by\": \"admin\",\n" +
+      "            \"zip\": \"\",\n" +
+      "            \"home_phone\": \"\",\n" +
+      "            \"time_format\": \"\",\n" +
+      "            \"last_login\": \"2019-04-05\",\n" +
+      "            \"active\": \"true\",\n" +
+      "            \"sys_domain_path\": \"/\",\n" +
+      "            \"cost_center\": \"\",\n" +
+      "            \"phone\": \"\",\n" +
+      "            \"name\": \"survey user\",\n" +
+      "            \"employee_number\": \"\",\n" +
+      "            \"gender\": \"\",\n" +
+      "            \"city\": \"\",\n" +
+      "            \"failed_attempts\": \"0\",\n" +
+      "            \"user_name\": \"survey.user\",\n" +
+      "            \"title\": \"\",\n" +
+      "            \"sys_class_name\": \"sys_user\",\n" +
+      "            \"sys_id\": \"005d500b536073005e0addeeff7b12f4\",\n" +
+      "            \"internal_integration_user\": \"false\",\n" +
+      "            \"ldap_server\": \"\",\n" +
+      "            \"mobile_phone\": \"\",\n" +
+      "            \"street\": \"\",\n" +
+      "            \"company\": \"\",\n" +
+      "            \"department\": \"\",\n" +
+      "            \"first_name\": \"survey\",\n" +
+      "            \"email\": \"survey.user@email.com\",\n" +
+      "            \"introduction\": \"\",\n" +
+      "            \"preferred_language\": \"\",\n" +
+      "            \"manager\": \"\",\n" +
+      "            \"sys_mod_count\": \"1\",\n" +
+      "            \"last_name\": \"user\",\n" +
+      "            \"photo\": \"\",\n" +
+      "            \"avatar\": \"\",\n" +
+      "            \"middle_name\": \"\",\n" +
+      "            \"sys_tags\": \"\",\n" +
+      "            \"time_zone\": \"\",\n" +
+      "            \"schedule\": \"\",\n" +
+      "            \"date_format\": \"\",\n" +
+      "            \"location\": \"\"]\n" +
+      "}";
+    RestAPIResponse restAPIResponse = new RestAPIResponse(httpStatus, headers, responseBody);
+    Mockito.when(restApi.executeGet(Mockito.any())).thenReturn(restAPIResponse);
+    Mockito.when(restApi.parseResponseToResultListOfMap(restAPIResponse.getResponseBody())).thenReturn(result);
+    serviceNowMultiSource.configurePipeline(mockPipelineConfigurer);
+    Assert.assertEquals(0, mockFailureCollector.getValidationFailures().size());
```

Comment:
Same comment

Reply:
Added the assertion for output schema as well. c0e6a0f